src/Apps/Artist/Routes/Overview/Components/ArtistArtworkFilter.tsx

```diff
@@ -65,7 +71,7 @@ const ArtistArtworkFilter: React.FC<ArtistArtworkFilterProps> = props => {
 }
 
 export const ArtistArtworkFilterRefetchContainer = createRefetchContainer(
-  ArtistArtworkFilter,
+  withRouter(ArtistArtworkFilter),
```

Comment:
This was a breaking change in 0.4 of `found`. Without this HOC, the `match` prop didn't seem to exist (and `location` also wasn't present as it was no longer spread out of `match` - https://github.com/relay-tools/found-relay/pull/163).

I still don't get why we now need `withRouter`, but that seems to be the documented way in 0.4.

Reply:
`withRouter` plucks the `match` value out of a React context, provided by the router, and then injects it as a prop into the wrapped component